src/main/resources/application.yml

```diff
@@ -2,12 +2,23 @@ api:
   key: ${CHS_API_KEY}
   url: ${API_URL}
 
+aws:
+  accessKeyId: ${AWS_ACCESS_KEY_ID}
+  region: ${ENV_REGION_AWS}
```

Comment:
Will `ENV_REGION_AWS` be added to docker, rebel1 and cidev? And I assume our own AWS keys will be read from our bash/zshrc profile?

Reply:
ENV_REGION_AWS is in docker-chs-development and already is in rebel1 and cidev